lib/logs.coffee

```diff
@@ -135,7 +135,7 @@ getLogs = (deps, opts) ->
 	#
 	# @param {String|Number} uuidOrId - device uuid (string) or id (number)
 	# @param {Object} [options] - options
-	# @param {Number} [options.count] - number of historical messages to include
+	# @param {Number} [options.count] - number of historical messages to include (or 'all')
```

Comment:
Should we also change the `{Number}` to `{Number|String}` (or even `{Number|'all'}` if it's allowed)?